### PR TITLE
doc: Add examples for X-ray dynamic tracing to live and record man pages

### PR DESCRIPTION
**Pull request by Taeung**
Currently uftrace supports the dynamic tracing feature 'X-ray'
of Clang/LLVM 4.0 so add examples for it to live and record man pages.

I referred to your mention on #88 😄 
If you give me some feedback, I'll update this PR.

### CODE REVIEW

File: doc/uftrace-live.md [hunk 1]
```diff
@@ -355,6 +355,28 @@ In addition, you can enable all functions at load time using '.' that matches to
        2.451 us [19387] |   } /* a */
        3.289 us [19387] | } /* main */
 
+Clang/LLVM 4.0 provides a dynamic tracing feature called [X-ray](http://llvm.org/docs/XRay.html).  It's similar to `gcc -mfentry -mnop-mcount` and `-finstrument-functions`.  The uftrace tool also supports `X-ray`.
```
Comment (namhyung) [inline doc/uftrace-live.md]: s/dynamic tracing feature/dynamic instrumentation technique/
s/similar to/similar to a combination of/

And the last sentence can be rewritten as:

The uftrace also supports dynamic tracing on the executables built with the `X-ray`.

File: doc/uftrace-record.md [hunk 2]
```diff
@@ -331,6 +331,30 @@ In addition, you can enable all functions at load time using '.' that matches to
        2.451 us [19387] |   } /* a */
        3.289 us [19387] | } /* main */
 
+Clang/LLVM 4.0 provides a dynamic tracing feature called [X-ray](http://llvm.org/docs/XRay.html).  It's similar to `gcc -mfentry -mnop-mcount` and `-finstrument-functions`.  The uftrace tool also supports `X-ray`.
```
Comment (namhyung) [inline doc/uftrace-record.md]: Ditto.

File: doc/uftrace-live.md (hunk 1 above)
Comment (Taeung) [inline doc/uftrace-live.md]: I got it ! will send PR v2 !

**Comment by Taeung** [comment]
@namhyung , I sent PR v2 based on your comment. 😄

**Pull request merged**